CHANGES.md

```diff
@@ -1,6 +1,462 @@
+  - made `cstdlib` atomics the default for the gcc, clang, and llvm back-ends
+  - introduced variants of RA that use unordered atomics and oversubscription
+  - added new primers for records, tuples, and special object methods
+
+Packaging / Configuration Changes
+---------------------------------
+* `CHPL_TARGET_ARCH` now refers to machine types and `CHPL_TARGET_CPU` to cpus
+  (e.g., `export CHPL_TARGET_ARCH=x86_64`, `CHPL_TARGET_CPU=sandybridge`
+   see https://chapel-lang.org/docs/1.19/usingchapel/chplenv.html#chpl-target-arch
+   and https://chapel-lang.org/docs/1.19/usingchapel/chplenv.html#chpl-target-cpu)
+* made `cstdlib` atomics the default for gcc, clang and llvm compilers
+  (see https://chapel-lang.org/docs//1.19/usingchapel/chplenv.html#chpl-atomics)
+
+Syntactic/Naming Changes
+------------------------
+* made `true`, `false`, and numeric type names into reserved words in Chapel
+  (see the 'Keywords' section in the 'Lexical Structure' chapter of the spec))
+* added support for underscores in integer and floating point literals
+  (e.g., `1_000_000` is now the same as `1000000`
+   see the 'Literals' section in the 'Lexical Structure' chapter of the spec))
+
+Semantic Changes / Changes to Chapel Language
+---------------------------------------------
+* made `throw` and `catch` operate on `owned` errors, enabling re-throwing
+  (see https://chapel-lang.org/docs/1.19/primers/errorHandling.html)
+* for records, compiler-generated `=`, `==`, `!=` now require matching types
+  (see 'Record Assignment' and 'Default Comparison Operators' in the spec)
+* `[]`-loops now permit serial execution if parallelism isn't an option
+  (see 'The Forall Statement' in the 'Data Parallelism' chapter of the spec))
+* made default string accesses always return a string
+  (see https://chapel-lang.org/docs/1.19/builtins/String.html#String.string.this)
+* default initializers for generic types require formal names to match fields
+  (see https://chapel-lang.org/docs/1.19/technotes/initTypeAlias.html)
+
+New Features
+------------
+* added support for directly initializing atomic variables from values
+  (e.g., `var x: atomic int = 10;`)
+* added lifetime annotations for expressing relative lifetimes between arguments
+  (see https://chapel-lang.org/docs/1.19/technotes/lifetimeChecking.html#lifetime-annotations)
+* certain dereferences of `nil` values are now caught by the compiler
+  (see https://chapel-lang.org/docs/1.19/technotes/nilChecking.html)
+* added support for using type aliases in `new` expressions
+  (see https://chapel-lang.org/docs/1.19/technotes/initTypeAlias.html)
+* added support for the `init=` method, a proposed change to copy initializers
+  (see https://chapel-lang.org/docs/1.19/technotes/initequals.html)
+* added support for ranges of codepoints and slicing of strings using them
+  (see https://chapel-lang.org/docs/1.19/builtins/String.html)
+* added string.codePoint[s]() to access and iterate over strings by codepoint
```

Comment:
`string.codePoint[s]()` and `string.byte[s]()` render weirdly as markdown (`[s]()` renders as a hyperlink). Wrap them with `` ?

Reply:
Good catch, thanks!